### PR TITLE
[MAIN] Update Copyright

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

File: tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
-#  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-#  the specific language governing rights and limitations under the License.
-#
-#  The Covered Software is Terracotta Core.
-#
-#  The Initial Developer of the Covered Software is
-#  Terracotta, Inc., a Software AG company
-#
-#
+ # Copyright Terracotta, Inc.
```
Comment (cljohnso) [inline tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties:1]: Does the Copyright item below need updating?

File: test-common/src/main/java/com/tc/util/Grep.java [hunk 2]
```diff
@@ -1,22 +1,20 @@
 /*
+ *  Copyright Terracotta, Inc.
```
Comment (cljohnso) [inline test-common/src/main/java/com/tc/util/Grep.java:2]: Should have a look at https://github.com/Terracotta-OSS/dso/blob/main/common/src/test/java/com/tc/util/Grep.java

File: tc-server/src/test/java/com/tc/net/protocol/transport/ServerStackProviderTest.java [hunk 3]
```diff
@@ -1,19 +1,18 @@
 /*
+ *  Copyright Terracotta, Inc.
```
Comment (cljohnso) [inline tc-server/src/test/java/com/tc/net/protocol/transport/ServerStackProviderTest.java:2]: Stray copyright statement below this block ...

File: tc-server/src/test/java/com/tc/classloader/ServiceLocatorTest.java [hunk 4]
```diff
@@ -1,19 +1,18 @@
 /*
+ *  Copyright Terracotta, Inc.
```
Comment (cljohnso) [inline tc-server/src/test/java/com/tc/classloader/ServiceLocatorTest.java:2]: Stray copyright below this block

File: test-common/src/main/java/com/tc/util/Grep.java (hunk 2 above)
Comment (myronkscott) [reply test-common/src/main/java/com/tc/util/Grep.java:2]: going to just remove the class.  Not used

File: tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties (hunk 1 above)
Comment (myronkscott) [reply tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties:1]: looks correct to me.  what do you see?

File: tc-server/src/test/java/com/tc/classloader/ServiceLocatorTest.java (hunk 4 above)
Comment (myronkscott) [reply tc-server/src/test/java/com/tc/classloader/ServiceLocatorTest.java:2]: will fix

File: tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties (hunk 1 above)
Comment (cljohnso) [reply tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties:1]: There's a line later in the file against which I cannot make a review comment 

```java
Copyright            = Internal use only; Terracotta copyright information
```

Is this one correct?

Comment (myronkscott) [reply tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties:1]: Yes.  Correct.  That is the descriptor of the attribute.

**Review by cljohnso — APPROVED** [review]

**Pull request merged**